src/include/binder/binder.h

```diff
@@ -1,10 +1,11 @@
 #pragma once
-
 #include "binder/query/bound_regular_query.h"
 #include "common/copier_config/copier_config.h"
 #include "expression_binder.h"
 #include "parser/query/regular_query.h"
 #include "query_normalizer.h"
+#include <arrow/api.h>
+#include <arrow/filesystem/api.h>
```

Comment:
I hope we don't introduce arrow headers in the header file, as this will be propagated to `main` module, too.
Can we add a forward declaration of `arrow::Status` instead? Or move the `throwExceptionIfNotOK` function to cpp file.

Reply:
#include <arrow/api.h>
#include <arrow/filesystem/api.h>

are moved to the CPP files.